math/number_of_digits.cpp

```diff
@@ -0,0 +1,24 @@
+#include<iostream>
+#include<math.h>
+
+unsigned int digits(unsigned int n) {
+    int num_digits=0;
+    while(n>0) {
+        num_digits++;
+        n=n/10;
+    }
+    return num_digits;
+}
+
+unsigned int digits_log(unsigned int n) {
+    return log10(n)+1;
+}
+
+int main() {
+	int num=12345;
+	std::cout << "Number of digits in " << num << " is : " 
+		<< digits(num) << std::endl;
+	//Efficient approach to find the number of digits of a number
```

Comment:
`math/number_of_digits.cpp:21:  Tab found; better to use spaces  [whitespace/tab] [1]
math/number_of_digits.cpp:21:  Should have a space between // and comment  [whitespace/comments] [4]`